src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/hmi_notifications_test.cc

```diff
@@ -332,57 +332,63 @@ class HMIOnNotificationsEventDispatcher
   NiceMock<event_engine_test::MockEventDispatcher> mock_event_dispatcher_;
 };
 
-typedef Types<OnVIAccPedalPositionNotification,
-              OnVIBeltStatusNotification,
-              OnVIBodyInformationNotification,
-              OnVIDeviceStatusNotification,
-              OnVIDriverBrakingNotification,
-              OnVIEngineTorqueNotification,
-              OnVIExternalTemperatureNotification,
-              OnVIFuelLevelNotification,
-              OnVIFuelLevelStateNotification,
-              OnVIGpsDataNotification,
-              OnVIHeadLampStatusNotification,
-              OnVIInstantFuelConsumptionNotification,
-              OnVIMyKeyNotification,
-              OnVIOdometerNotification,
-              OnVIPrndlNotification,
-              OnVIRpmNotification,
-              OnVISpeedNotification,
-              OnVISteeringWheelAngleNotification,
-              OnVITirePressureNotification,
-              OnVIVehicleDataNotification,
-              OnVIVinNotification,
-              OnVIWiperStatusNotification> HMIOnViNotificationsTypes;
-
-typedef Types<OnAppPermissionChangedNotification,
-              OnAudioDataStreamingNotification,
-              hmi::OnButtonSubscriptionNotification,
-              OnFileRemovedNotification,
-              OnPutFileNotification,
-              OnResumeAudioSourceNotification,
-              OnSDLCloseNotification,
-              OnSDLConsentNeededNotification,
-              OnSDLPersistenceCompleteNotification,
-              OnStatusUpdateNotification,
-              OnVideoDataStreamingNotification,
-              OnRecordStartdNotification> HMIOnNotificationsListToHMITypes;
+typedef Types<sdl_rpc_plugin::commands::OnVIAccPedalPositionNotification,
```

Comment:
@ValeriiMalkov I believe you could just replace at line 161
```
using namespace am::commands;
```
with 
```
using namespace sdl_rpc_plugin::commands;
```
and then you wouldn't need to specify full namespace path explicitly in all of your code below.
Please check that.